src/server/routes/topics.ts

```diff
@@ -0,0 +1,32 @@
+import express from 'express'
+import type { Response, Request } from 'express'
+import { getTopics } from '../../utils/db'
+import { ObjectId } from 'mongodb'
+
+const topics = express.Router()
+
+topics
+  .get('/many', async (req: Request, res: Response) => {
```

Comment:
I think just `/` is sufficient, since you're already on `/topics` 
checkout [this site](https://restfulapi.net/rest-api-design-tutorial-with-example/) for more